Matlab/Miscellaneous.tmPreferences

```diff
@@ -34,21 +34,24 @@
 		<array>
 			<dict>
 				<key>name</key>
-				<string>TM_COMMENT_START</string>
+				<string>TM_COMMENT_START_2</string>
```

Comment:
To fixed incompatibility with the Octave theme file extension:
https://github.com/evandrocoan/SublimeTextStudio/tree/f9c1f9143126c9f37a1ed0a0a623392d2c2f81eb/Octave

The `%` comments were overriding the Octave's `#` comments. 

If you see my octave syntax extend the Matlab's one, adding missing scopes. As the `#` comments type. So I like to use `#` for octave and `%` Matlab. But if I set the Matlab `TM_COMMENT_START_2` as `TM_COMMENT_START`, Sublime Text to start using `%` on my Octave files.

This as the combiantion which worked perfectly, i.e., to keep the Matlab using `%` for line/block comments, and Octave's to keep using `#` for line/block comments:

Octave/Miscellaneous.tmPreferences
```
...
		<array>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START</string>
				<key>value</key>
				<string># </string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START_1</string>
				<key>value</key>
				<string>#{
</string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_END_1</string>
				<key>value</key>
				<string>
#}
</string>
			</dict>
		</array>
...
```

Matlab/Miscellaneous.tmPreferences
```
...
		<array>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START_2</string>
				<key>value</key>
				<string>% </string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_START_3</string>
				<key>value</key>
				<string>%{
</string>
			</dict>
			<dict>
				<key>name</key>
				<string>TM_COMMENT_END_3</string>
				<key>value</key>
				<string>
%}
</string>
			</dict>
		</array>
...
```